nf_core/pipelines/create/create.py

```diff
@@ -182,7 +182,7 @@ def update_config(self, organisation, version, force, outdir):
             self.config.force = force if force else False
         if self.config.outdir is None:
             self.config.outdir = outdir if outdir else "."
-        if self.config.is_nfcore is None:
+        if self.config.is_nfcore is None or self.config.is_nfcore == "null":
```

Comment:
is this a yaml parsing thing, that it turns into `"null"`?

Reply:
yes, all `None` values are written as `null`